packages/core/helper-plugin/src/utils/getFetchClient/index.js

```diff
@@ -1,23 +1,34 @@
 import instance from '../fetchClient';
+import qs from 'qs';
 
 const addPrependingSlash = (url) =>
   typeof url === 'string' && url.charAt(0) !== '/' ? `/${url}` : url;
 
-//This regular expression matches a string that starts with either "http://" or "https://" or any other protocol name in lower case letters, followed by "://" and ends with anything else
+// This regular expression matches a string that starts with either "http://" or "https://" or any other protocol name in lower case letters, followed by "://" and ends with anything else
 const hasProtocol = (url) => new RegExp('^(?:[a-z+]+:)?//', 'i').test(url);
 
-// check if the url has a prepending slash, if not it adds the slash
+// Check if the url has a prepending slash, if not add a slash
 const normalizeUrl = (url) => (hasProtocol(url) ? url : addPrependingSlash(url));
 
+const paramsSerializer = (params) => {
+  return qs.stringify(params, { encode: false });
+};
+
 const getFetchClient = (defaultOptions = {}) => {
   instance.defaults.baseURL = window.strapi.backendURL;
   return {
-    get: (url, config) => instance.get(normalizeUrl(url), { ...defaultOptions, ...config }),
+    get: (url, config) =>
+      instance.get(normalizeUrl(url), {
+        ...defaultOptions,
+        ...config,
+        paramsSerializer,
```

Comment:
I think(?) This might be able to be applied to the `instance`?

Reply:
Yup sure can! Makes way more sense, thanks 👍

Reply:
amazing.